packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -617,22 +618,80 @@ class _ConstraintsDescriptorState extends State<ConstraintsDescriptor>
 
 
 class StoryOfYourFlexWidget extends StatelessWidget {
-  const StoryOfYourFlexWidget(InspectorTreeNode node, {
+  StoryOfYourFlexWidget(this.node, {
     Key key,
   }) : super(key: key);
 
+  InspectorTreeNode node;
+
   @override
   Widget build(BuildContext context) {
+    final Map<String, Object> flexDetails = node.diagnostic?.json['flex'];
+    final int numOfChildren = node.diagnostic.childrenNow.length;
+    final List<Widget> children = node.diagnostic.childrenNow.map((RemoteDiagnosticsNode child) => Expanded(
+      flex: 1,
```

Comment:
you can now use List comprehensions for this sort of case in dart. so
[for (var child in node.diagnostic.childrenNow) 
   Container(something something)
];